### DISCUSSION TITLE
Exact match for chat trigger

### DISCUSSION THREAD

**Discussion by meza** [gameserverapp/Platform#446] [Suggestions]
Currently there's the `contains` trigger which checks if the message contains a given word. The issue with this is that when people are trying to help out fellow players by telling them about a command, they end up triggering it themselves.

I'd like to suggest an `exact` trigger that only kicks in when the message is an exact match for the given words.

**Comment by mvaessen** [comment]
see #447

**Comment by mvaessen** [comment]
This is now available on the dashboard!

**Comment by meza** [comment]
This is amazing, thank you!
